src/wp-admin/includes/class-walker-nav-menu-edit.php

```diff
@@ -145,15 +148,31 @@ public function start_el( &$output, $item, $depth = 0, $args = array(), $id = 0
 										remove_query_arg( $removed_args, admin_url( 'nav-menus.php' ) )
 									),
 									'move-menu_item'
+								),
+								esc_attr__( 'Move down' )
 								);
 							?>
-							" class="item-move-down" aria-label="<?php esc_attr_e( 'Move down' ); ?>">&#8595;</a>
 						</span>
-						<a class="item-edit" id="edit-<?php echo $item_id; ?>" href="
 																 <?php
-																	echo ( isset( $_GET['edit-menu-item'] ) && $item_id == $_GET['edit-menu-item'] ) ? admin_url( 'nav-menus.php' ) : add_query_arg( 'edit-menu-item', $item_id, remove_query_arg( $removed_args, admin_url( 'nav-menus.php#menu-item-settings-' . $item_id ) ) );
+						if ( isset( $_GET['edit-menu-item'] ) && $item_id == $_GET['edit-menu-item'] ) {
+							$edit_url = admin_url( 'nav-menus.php' );
+						} else {
+							$edit_url = add_query_arg(
+								array(
```

Comment:
Why did this change to `array`?

Reply:
This was part of 45614, both ways are legitimate uses of [add_query_arg()](https://developer.wordpress.org/reference/functions/add_query_arg/) with the array method being more extensible.

Reply:
But this is a coding standards PR, and the `array` isn't in the original and when it's a single value it doesn't have to be on multiple lines, etc.